pkg/doc/jose/jwk.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package jose
+
+import "github.com/square/go-jose/v3"
+
+// JWK (JSON Web Key) is a JSON data structure that represents a cryptographic key.
+type JWK jose.JSONWebKey
+
+// JWK gets JWK from JOSE headers.
+func (h Headers) JWK() (*JWK, bool) {
+	jwkRaw, ok := h[HeaderJSONWebKey]
+	if !ok {
+		return nil, false
+	}
+
+	var jwk JWK
+
+	err := convertMapToValue(jwkRaw, &jwk)
+	if err != nil {
+		return nil, false
+	}
+
+	return &jwk, true
+}
+
+// AriesJWK formatted key
+type AriesJWK struct {
```

Comment:
I'm a bit surprised to see an `Aries` prefix in this package. What makes this struct Aries-specific?

Reply:
That was just a temporary name I gave it in order to avoid a type name collision - it will be removed :)

Reply:
yeah.. remove this struct.. it's not needed, we should only use JWK type at line 12 above

Reply:
Done